### DISCUSSION TITLE
disable cuda warning

### DISCUSSION THREAD

**Discussion by the1gofer** [explosion/spaCy#10587] [Help: Coding & Implementations] [answered]
I'm running the en_core_web_trf on a Mac M1, and get the following:
`.../venv/lib/python3.9/site-packages/torch/autocast_mode.py:162: UserWarning: User provided device_type of 'cuda', but CUDA is not available. Disablingt]
  warnings.warn('User provided device_type of \'cuda\', but CUDA is not available. Disabling')
`

I've seen some poeple suggest i use the lg model, but with some testing the trf seems far more accurate for my use.  I don't really care if it's slow, but it seems to work other than the warning.  Can I disable it?   I have tried the following, and it doens't seem to work.

``` 
with warnings.catch_warnings():
    warnings.simplefilter("ignore")
    self.doc = self.nlp(obit_text)
```

**Comment by the1gofer** [comment]
Any thoughts?

**Comment by ljvmiranda921** [comment, marked as answer]
If you just wish to disable warnings, you can try:

```python
import warnings
warnings.filterwarnings("ignore", category=UserWarning)
```

Although it's possible that the warning happens during import, not execution, that's why it still showed up in your initial approach.

**Reply by the1gofer** [reply]
Thanks, I'll see if that works.

**Reply by sushant1253** [reply]
This works, Thanks.

**Comment by the1gofer** [comment]
That seems to have done the trick.  Thank you.

Jason Crews
